modules/imgproc/src/canny.cpp

```diff
@@ -805,6 +865,20 @@ void Canny( InputArray _src, OutputArray _dst,
 
     Mat src = _src.getMat(), dst = _dst.getMat();
 
+    CV_OVX_RUN(
+        false && /* disabling due to accuracy issues */
```

Comment:
But generally I agree with @alalek: the `Imgproc_Canny.accuracy` and  `Imgproc_Canny.accuracy_deriv` are **very bad tests**!
I'm going to disable them and add another one - comparing VX vs non-VX results...